### DISCUSSION TITLE
Failed Gradle action duplicates log

### DISCUSSION THREAD

**Discussion by bendezu** [fastlane/fastlane#21836] [Q&A]
I am having trouble running the gradle action in fastlane. When the action fails, the terminal output is excessively long due to the gradle logs being duplicated three times. I have tried to find information on how to disable this verbose output, but have been unsuccessful. 
Here is an example of the output I receive when launching `bundle exec fastlane gradleTest`. `gradleTest` is a lane that simply calls `gradle(task: 'unexistedTask')`.
[gradle.log](https://github.com/fastlane/fastlane/files/14088415/gradle.log)

Thanks for your help!

**Comment by amidaleet** [comment]
[This fix PR](https://github.com/fastlane/fastlane/pull/29525) seems to be related
